demos-europe/demosplan-core
# chore: (refs DPLAN-12315): Replace vue draggable with vue sortable in adminlayerList

### Ticket
DPLAN-12315

Rework AdminLayerList to work with useSortable. That had to be replaced because draggable doesn't work with vue3 anymore. 
Since the logic/concepts of draggable are quite different, it had to be changed a lot.

this replaces https://github.com/demos-europe/demosplan-core/pull/2474 in the way that te old PR had to be rework too much to continue





### How to review/test


As Fachplaner Goto: /verfahren/<procedure-id>/verwalten/gislayer

Create, remove sort, resort layers and categories in "reihenfolge für Navigation" and "reihenfolge auf Karte". Save it dismiss it, check if its correct on public detail.
everything should work as before.

### Needs to be merged first:

- [ ] https://github.com/demos-europe/demosplan-ui/pull/682
